JavaScript/JavaScript.sublime-syntax

```diff
@@ -15,6 +15,22 @@ variables:
   func_lookahead: '\s*\b(async\s+)?function\b'
   arrow_func_lookahead: '\s*(\basync\s*)?([_$[:alpha:]][_$[:alnum:]]*|\(([^()]|\([^()]*\))*\))\s*=>'
 
+  method_name: >-
+    (?x)(?:
+      {{identifier}}
+      | '(?:[^\\']|\\.)*'
+      | "(?:[^\\"]|\\.)*"
+      | \[ {{identifier}} (?:\.{{identifier}})* \]
+    )
+
+  line_continuation_lookahead: >-
+    (?x)
+    (?! \+\+ | -- )
+    (?=
+      != |
+      [ -+*/% ><= &|^ \[( ;,.:? ]
+    )
```

Comment:
Could you explain why this `line_continuation_lookahead` exists? Why can't you check for a backslash followed by a newline character?